### PR TITLE
travis support

### PR DESCRIPTION
**Pull request by ateshuseyin**
This PR adds travis support by adding .travis.yml file. Not ready yet. This is an early request for early reviews. 
@spatialillusions You can login to https://travis-ci.org by github account and activate this repo. That's it.

### CODE REVIEW

**Comment by ateshuseyin** [comment]
Test is very long process. That causes below error:

> No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.

> Check

 the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+language: node_js
+node_js:
+  - "8"
+script:
+  - npm run build
+  - npm run bundle
```
Comment (okwolf) [inline .travis.yml]: Shouldn't we do `npm run test` at some point?

Comment (ateshuseyin) [reply .travis.yml]: You are right @okwolf but it is already part of prebuild step https://github.com/spatialillusions/milsymbol/blob/master/package.json#L12

Comment (okwolf) [reply .travis.yml]: @ateshuseyin an that note, isn't `bundle` already part of [`build`](https://github.com/spatialillusions/milsymbol/blob/master/package.json#L13)?

Comment (ateshuseyin) [reply .travis.yml]: Oooww. You are right. `npm run build` is only command we need it. Thanks

**Comment by ateshuseyin** [comment]
PASS  test/ms.test.js (750.263s)
  ● Console
    console.info src/ms.js:237
      milsymbol.js 2.0.0-master - Copyright (c) 2017 Måns Beckman  http://www.spatialillusions.com
No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received

Fails because of no output for 10 minutes. I cannot fix it. Any help is welcome.

**Comment by ateshuseyin** [comment]
relates #180

**Pull request merged**